build script tries to install NTL in /usr/local/include/NTL

When executing `Pkg.build("Singular")`, the build script seems to try to install NTL under `/usr/local/include/NTL`, which usually fails when executed as a non-privileged user.

```
make[1]: Leaving directory '/home/florian/.julia/v0.6/Singular/deps/tmpEConDD/ntl-9.3.0/src'
touch all
mkdir -p -m 755 /usr/local/include
rm -rf /usr/local/include/NTL
mkdir -m 755 /usr/local/include/NTL
mkdir: cannot create directory ‘/usr/local/include/NTL’: Permission denied
makefile:427: recipe for target 'install' failed
make: *** [install] Error 1
===============================================[ ERROR: Singular ]================================================

LoadError: failed process: Process(`make install`, ProcessExited(2)) [2]
while loading /home/florian/.julia/v0.6/Singular/deps/build.jl, in expression starting on line 30

==================================================================================================================

=================================================[ BUILD ERRORS ]=================================================

WARNING: Singular had build errors.

 - packages with build errors remain installed in /home/florian/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("Singular")`
 - build a single package by running its `deps/build.jl` script

==================================================================================================================

```

--- a/deps/build.jl
+++ b/deps/build.jl
@@ -18,7 +18,7 @@ cd(wdir)
 
 # INSTALL NTL
 
-const ntl="ntl-9.3.0"
+const ntl="ntl-10.5.0"
 
 try
   run(`wget -q -nc -c -O "$wdir/$ntl.tar.gz" "http://www.shoup.net/ntl/$ntl.tar.gz"`)

